### PR TITLE
added the code for the registration page stickiness + resolved null values for page

### PR DESCRIPTION
**Pull request by chasehammock**

### CODE REVIEW

File: src/main/webapp/WEB-INF/register.jsp [hunk 1]
```diff
@@ -9,27 +9,65 @@
...
-    first_name = request.getParameter("first_name");
-    last_name = request.getParameter("last_name");
-    email = request.getParameter("email");
-    street_address = request.getParameter("street_address");
-    zip_code = request.getParameter("zip_code");
-    state = request.getParameter("state");
-    phone = request.getParameter("phone");
-    password = request.getParameter("password");
-    confirm_password = request.getParameter("confirm_password");
+<%  String username_temp=request.getParameter("username");
```
Comment (MontealegreLuis) [inline src/main/webapp/WEB-INF/register.jsp]: This code (lines 12 to 70) should be in your Servlet. Please move it to the Servlet and test your changes again.

**Review by MontealegreLuis — DISMISSED** [review]
Move the code in the jsp file to your servlet please

**Comment by chasehammock** [comment]
Moved the .jsp java code to the Registration servlet as requested. Thanks Luis! 👍

**Comment by chasehammock** [comment]
didn't create a new branch for the login servlet and jsp changes, and the validation messages in login were also added to the changes for this branch.

**Comment by MontealegreLuis** [comment]
Thank you!

**Pull request merged**